base/applications/cleanmgr/choicepage.c

```diff
@@ -0,0 +1,135 @@
+/*
+ * PROJECT:         ReactOS Disk Cleanup
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * PURPOSE:         Child dialog proc
+ * COPYRIGHT:       Copyright 2020 Arnav Bhatt (arnavbhatt288 at gmail dot com)
+ */
+
+#include "resource.h"
+#include "precomp.h"
+
+DIRSIZE sz;
+WCHAR_VAR wcv;
+BOOL_VAR bv;
+
+INT_PTR CALLBACK ChoicePageDlgProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lParam)
+{
+	WCHAR tempList[MAX_PATH] = { 0 };
+	WCHAR stringText[MAX_PATH] = { 0 };
+	WCHAR fullText[MAX_PATH] = { 0 };
+	WCHAR totalAmount[MAX_PATH] = { 0 };
+	static WCHAR* viewFolder = NULL;
+	uint64_t tempSize = sz.tempASize + sz.tempBSize;
+	uint64_t recycleSize = sz.recyclebinSize;
+	uint64_t downloadSize = sz.downloadedSize;
+	uint64_t rappsSize = sz.rappsSize;
+	static HWND hList = 0;
+	
+	LVCOLUMNW lvC;
+	ZeroMemory(&lvC, sizeof(lvC));
```

Comment:
This code make procedure execution slower. Move these initialization code to the specific place.